cdk/visit/lambda_code/register_user/register_user.py

```diff
@@ -103,8 +105,9 @@ def add_user_info(self, user_info):
                 'last_name': {'S': user_info['lastName']},
                 'gender': {'S': user_info['Gender']},
                 'date_of_birth': {'S': user_info['DOB']},
+                'position': {'S': user_info['UserPosition']},
                 'grad_semester': {'S': grad_sem},
-                'grad_year': {'N': grad_year},
+                'grad_year': {'S': grad_year},
```

Comment:
What is the reason for representing this as a string? Bit of a bad practice unless you have a specific reason for it. The reason being, strings require lexicographical sorting and numbers require numerical sorting. Shouldnt really impact us here, but just curious what the rationale was

Reply:
I made it a string because the registration would fail whenever the year was a number and was left blank by a faculty/staff role, but changing it to a string got it to work.
I'm not sure if my recent change would fix that issue, but I hadn't thought of changing it back.